sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -27,16 +31,9 @@
 #### Other Changes
 * Added coverage for `ChangeFeedProcessor` in `Latest Version` change feed mode to read change feed from a custom start time for multi-write accounts. - See[PR 36257](https://github.com/Azure/azure-sdk-for-java/pull/36257)
 
-### 4.48.2 (2023-08-25)
```

Comment:
why this change

Reply:
Faulty merge conflict resolution. Fixing this in next iteration.